tools/benchmark/cmd/mvcc-put.go

```diff
@@ -106,16 +108,25 @@ func mvccPutFunc(cmd *cobra.Command, args []string) {
 	rrc := r.Results()
 
 	rc := r.Run()
-	for i := 0; i < totalNrKeys; i++ {
-		st := time.Now()
-		if txn {
+
+	if txn {
+		for i := 0; i < totalNrKeys; i += nrBatch {
+			st := time.Now()
+
 			tw := s.Write()
-			tw.Put(keys[i], vals[i], lease.NoLease)
+			for j := i; j < i+nrBatch; j++ {
+				tw.Put(keys[j], vals[j], lease.NoLease)
+			}
 			tw.End()
-		} else {
+
+			rrc <- report.Result{Start: st, End: time.Now(), Ops: uint(nrBatch)}
```

Comment:
I added the field because I'm interested in the number of written keys in a unit of time. The motivation is evaluating how batched transactional write can contribute to the throughput in terms of the number of keys.

But I agree with you that the change introduces complexity a little bit. Should I separate the reporting feature for batch oriented benchmarks?